thunder/torch/__init__.py

```diff
@@ -1786,6 +1786,32 @@ def _reduction(
     return result
 
 
+@torchsymbol(torch.all, is_method=True, id="torch.all")
+def all_tensor(
```

Comment:
Let's also update the PyTorch executor with direct lowerings for both `all` and `any`